src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/client/ChipsRestClient.java

```diff
@@ -12,7 +13,8 @@
 @Service
 public class ChipsRestClient {
 
-    private static final String CHIPS_REST_ENDPOINT_URI_VAR = "{chips-rest-endpoint}";
+    @Value("${chips.rest.endpoint}")
+    private String chipsRestEndpoint;
```

Comment:
the rest endpoint will need to be provided in the incoming message, this looks like it is being set to the value 'endpoint' from the application.properties.

Reply:
Changed this back.